drizzlepac/hlautils/hla_flag_filter.py

```diff
@@ -231,38 +231,40 @@ def ci_filter(drizzled_image, catalog_name, catalog_data, proc_type, param_dict,
     ci_upper_limit = float(param_dict['quality control']['ci filter'][proc_type]['ci_upper_limit'])
     snr = float(param_dict['quality control']['ci filter'][proc_type]['bthresh'])
 
```

Comment:
```suggestion
par_dict = param_dict['quality control']['ci filter'][proc_type]
```
This definition would simplify the rest of the code by making it easier to read what is being accessed.  This is purely a suggestion for readability(and perhaps maintainability) which will NOT affect the computations and NOT something that needs to ever be changed.